### PR TITLE
gen-index.py: Fix APU6 version in generated entry

### PR DESCRIPTION
**Pull request by Sakoram**

### CODE REVIEW

**Comment by miczyg1** [comment]
@artur-rs sorry to ask for review, but you are the only editor of this file ATM. :)

**Review by miczyg1 — APPROVED** [review]
Let's wait for @artur-rs approval

**Review by artur-rs — APPROVED** [review]

**Pull request merged**
